### DISCUSSION TITLE
Post connection script / command

### DISCUSSION THREAD

**Discussion by Zibbp** [qdm12/gluetun#1205] [Q&A]
Hi,
Is there a way to run a script or command after a successful VPN connection is established?

**Comment by bnhf** [comment]
@Zibbp 

What about using the built-in Gluetun Http Control Server and scripting based on the output of an HTTP GET? You should be able to use either ``` /v1/openvpn/status ``` or ``` /v1/publicip/ip ```, and pattern match to determine the status of your OpenVPN connection. The script itself could run pretty much anywhere on your network.

https://github.com/qdm12/gluetun/wiki/HTTP-control-server
